docs/DeveloperGuide.adoc

```diff
@@ -121,7 +121,7 @@ The `**UI**` component,
 
 [[fig-LogicClassDiagram]]
 .Structure of the Logic Component
-image::LogicClassDiagram.png[]
+image::LogicClassDiagramv1.png[]
```

Comment:
The Logic Class Diagram is nicely done, there are no lines overlapping, the relationship can be seen clearly. 

How about changing colour scheme for boxes of different packages (eg. Red for class boxes of Parser package, Green for class boxes of Command package)? I think that will make the diagram even nicer and easier to see. 👍 

Also, is it necessary to add 2 exactly same Class Diagrams close to each other (Figure 2, Figure 5, Logic Class Diagram)?